curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-262-mountain-range.md

```diff
@@ -8,37 +8,39 @@ dashedName: problem-262-mountain-range
 
 # --description--
 
-The following equation represents the continuous topography of a mountainous region, giving the elevation h at any point (x,y):
+The following equation represents the continuous topography of a mountainous region, giving the elevation $h$ at any point ($x$,$y$):
 
-A mosquito intends to fly from A(200,200) to B(1400,1400), without leaving the area given by 0 ≤ x, y ≤ 1600.
+$$h = \left(5000 - \frac{x^2 + y^2 + xy}{200} + \frac{25(x + y)}{2}\right) \times e^{-\left|\frac{x^2 + y^2}{1\\,000\\,000} - \frac{3(x + y)}{2000} + \frac{7}{10}\right|}$$
```

Comment:
Small suggestion if we want to show the comma here:

```suggestion
$$h = \left(5000 - \frac{x^2 + y^2 + xy}{200} + \frac{25(x + y)}{2}\right) \times e^{-\left|\frac{x^2 + y^2}{1{,}000{,}000} - \frac{3(x + y)}{2000} + \frac{7}{10}\right|}$$
```

`\\,` didn't render the comma on my end, and when I tried `\,` or `,` alone, there was a space after the comma. Turns out it's because MathJax treats commas as punctuation for things like list entries by default.

A workaround is to wrap comma separators in brackets as mentioned here: https://github.com/mathjax/MathJax/issues/1615#issuecomment-245062032

Not necessary at all, but I thought it was pretty interesting.

Or if we'd like to be extra thorough:

```suggestion
$$h = \left(5{,}000 - \frac{x^2 + y^2 + xy}{200} + \frac{25(x + y)}{2}\right) \times e^{-\left|\frac{x^2 + y^2}{1{,}000{,}000} - \frac{3(x + y)}{2{,}000} + \frac{7}{10}\right|}$$
```

But these are just suggestions. I've looked through some of the previous Project Euler problems here and on projecteuler.net and it's a bit inconsistent. Maybe it's a bit like Rosetta Code in that contributors help create the different problems, and there are several different ways to group digits.

I'm fine with with sticking to the original text from Project Euler, though it is nice to separate larger digits like 1,000,000 as you did in the formula here.

Reply:
`{,}` is pretty nice, and it looks cleaner in MathJax command than `\\,`. I'd probably go with it from the start if I knew that works. When I was looking for a way to handle separating digits in large numbers in MathJax I've found https://math.meta.stackexchange.com/questions/5020/mathjax-basic-tutorial-and-quick-reference which in *Decimal Separator & Digit Separation* section of *Units* recommends using `\\,` There isn't anything more to it :stuck_out_tongue:. I don't really have any preference regarding this, I'm fine with whichever way is more wanted.

Reply:
Awesome, thanks for sending that link over. I wish I had found that when I was first working with MathJax and the early Project Euler problems :slightly_smiling_face:

In that case, let's stick with the suggested convention from that tutorial. As far as I can tell, whether or not to render the comma in formulas is largely up to personal taste and guidelines set by publications / universities. MathJax still inserts a subtle space there, so it should still help.

I don't have much of a preference either. Worst case scenario we can go back and add commas later with `{,}`.

Reply:
is there anything else to discuss here?

Reply:
resolving this conversation is the only thing left before the automatic merge